python/cudf_polars/.coveragerc

```diff
@@ -0,0 +1,3 @@
+# Configuration file for Python coverage tests
+[run]
+source = cudf_polars
```

Comment:
We already do [some configuration](https://github.com/rapidsai/cudf/blob/3420c71cb72f63db8d63164446cca042f354a08e/python/cudf_polars/pyproject.toml#L60-L68) of coverage in pyproject.toml. Can this configuration also go there (I suppose in a `[tool.coverage.run]` section?

Reply:
We may not even need a separate section for this. I pushed 1a2be0d to try using `pyproject.toml`.